public/content/documentation/web/component/autocomplete.md

```diff
@@ -0,0 +1,156 @@
+## General Notes
+
+How to test an autocomplete input with listbox
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab: Focus moves visibly to the text input
+   - Arrow-keys: Focus moves to and selects the next option. If the textbox is empty and the listbox is not displayed, opens the listbox and moves visual focus to the next option. In both cases DOM focus remains on the textbox.
+   - Enter: The textbox value is set to the content of the selected option. The listbox closes.
+   - Escape: Clears the textbox. If the listbox is displayed, closes it.
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the input, traverses list
+   - Doubletap: Opens select, chooses option
+
+3. Listen to screenreader output on all devices
+
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a select, popup, menu/submenu, listbox or combobox
+   - Group: Its label is read and selected options are read
+   - State: It indicates the value of the text input
+     
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+GIVEN THAT I am on a page with a header landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
```

Comment:
I'm not sure that we need this line, but you're welcome to leave it in if you feel strongly about it!